app/main.py

```diff
@@ -1,3 +1,2 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    return phrase.lower().count(letter.lower())
```

Comment:
The function is annotated to return an `int`, but the `str.count()` method already returns an integer. The code is correct, but ensure that the return type annotation matches the actual return type, which is indeed `int`.